### PR TITLE
adds Artist Studio blurb

### LINKED ISSUE #683: Add Artist Open studio to Spaces
Here is the approved content for the experience- Artist Open studio @edrushka 
Submitted by Angela -with request for Kevin to copy edit @KevZawacki

Artist Open Stuio
 Decription text  -Ensuring the health of the internet requires a diversity of approaches and voices. Art and design play key roles in engaging new audiences around these social issues and in shaping critical thinking around an open and healthy web. To highlight these artistic perspectives, MozFest, in collaboration with Digital Learning at V&A and Tate, has invited 5 artists to address each of the 5 thematic issue areas of the festival through an installation and on-site lab:

An Xiao Mina (Web Literacy)
Paolo Cirio (Privacy and Security)
Archana Prasad (Decentralization)
Gretta Louw (Open Innovation)
Brooklyn Pakathi (Digital Inclusion)

Participants will be able to engage with the artists in an open lab setting where they will be sharing their artistic practices and developing collaborative work with the public

### PR DESCRIPTION
**Pull request by hannahkane**
fixes #683 

Adds Artist Studio blurb, per #683

### CODE REVIEW

File: pages/spaces.jsx [hunk 1]
```diff
@@ -114,7 +114,16 @@ var ExpereincesInfo = [
     name: "Artist Open Studio",
     description:
     (<div>
-      <p></p>
+      <p>Upholding a healthy internet requires diverse voices and approaches—like art and design.</p>
```
Comment (mmmavis) [inline pages/spaces.jsx]: let's replace `-` with `&mdash;`

File: pages/spaces.jsx [hunk 2]
```diff
@@ -114,7 +114,16 @@ var ExpereincesInfo = [
...
+      <p>Art and design play key roles in engaging new audiences around issues like privacy and inclusion, and in shaping critical thinking around a healthy web. To highlight these artistic perspectives, MozFest—in collaboration with Digital Learning at V&A and Tate—has invited artists to address each of the five thematic issue areas of the festival through an installation and on-site lab:</p>
```
Comment (mmmavis) [inline pages/spaces.jsx]: same here, use `&mdash;` instead of `-`

File: pages/spaces.jsx [hunk 3]
```diff
@@ -114,7 +114,16 @@ var ExpereincesInfo = [
...
+        <ul>
+          <li>An Xiao Mina (Web Literacy)</li>
+          <li>Paolo Cirio (Privacy and Security)</li>
+          <li>Archana Prasad (Decentralization)</li>
+          <li>Gretta Louw (Open Innovation)</li>
+          <li>Brooklyn Pakathi (Digital Inclusion)</li>
+        </ul>
+      <p>Participants will engage with the artists in an open lab setting. Artists will share their practices, and also develop collaborative work with the public.</p>
```
Comment (mmmavis) [inline pages/spaces.jsx]: Let's don't wrap `<ul>` in `<p>`.  

https://www.w3.org/TR/html401/struct/text.html#h-9.3.1
> The P element represents a paragraph. It cannot contain block-level elements (including P itself).

Comment (hannahkane) [inline pages/spaces.jsx]: thanks, @mmmavis. Can you help me figure out what to do here? Should I use something other than `<ul>`?

Comment (mmmavis) [inline pages/spaces.jsx]: Oops! Got tricked by the indentation! My bad!

I think it's just indentation changes needed here:

```jsx
<p>Art and design play key roles in engaging new audiences around issues like privacy and inclusion, and in shaping critical thinking around a healthy web. To highlight these artistic perspectives, MozFest—in collaboration with Digital Learning at V&A and Tate—has invited artists to address each of the five thematic issue areas of the festival through an installation and on-site lab:
</p>
<ul>
  <li>An Xiao Mina (Web Literacy)</li>
  <li>Paolo Cirio (Privacy and Security)</li>
  <li>Archana Prasad (Decentralization)</li>
  <li>Gretta Louw (Open Innovation)</li>
  <li>Brooklyn Pakathi (Digital Inclusion)</li>
</ul>
```

**Comment by hannahkane** [comment]
@mmmavis - ok, I tried to address those issues. Can you take another look, please?

**Comment by mmmavis** [comment]
@hannahkane looks great. Thanks! 💐 🍾

**Pull request merged**